horace_core/sqw/@data_sqw_dnd/data_sqw_dnd.m

```diff
@@ -1,39 +1,25 @@
-classdef data_sqw_dnd < axes_block
-    % Class defines structure of the data, used by sqw&dnd objects
-    %
-    % Trivial implementation, wrapping around a structure
-
-    % Original author: T.G.Perring
+classdef data_sqw_dnd < DnDBase
+    % Transient class used as part of sqw object in the Horace, version < 4.0
```

Comment:
according to the class diagram its part of the dndbase object

Reply:
it was part of dndbase object. Not any more. description changed, class diagram will follow